src/quick-lint-js/cli/main.cpp

```diff
@@ -59,7 +59,8 @@ bool stderr_supports_terminal_escapes() {
 #if defined(_WIN32)
   return false;
 #else
-  return isatty(STDERR_FILENO);
+  const char *term = std::getenv("TERM");
+  return isatty(STDERR_FILENO) && term && term != "dumb"sv;
```

Comment:
Your code currently treats unset `TERM` differently than empty `TERM`. (Unset returns false; empty returns true.) Is that intentional?

Reply:
yes, that's what the gcc reference does, if TERM is set, and is not equal to "dumb", then it assumes to be able to emit escape sequences (at least on posix systems), it could be more accurate by looking up terminfo databases, but i feel that's too much trouble for little gain.